scripts/xe-backup-metadata

```diff
@@ -134,11 +156,25 @@ function cleanup {
    fi
 }
 
+# if we can't validate the UUID of the VDI, prompt the user
+if [ -n "${vdi_uuid}" ]; then
+    if ! validate_vdi_uuid "${sr_uuid}" "${vdi_uuid}"; then
+        local version="8.4"
+        echo "Backup VDI $vdi was created prior to XenServer $version."
```

Comment:
We should avoid the version number and name XenServer here.

Reply:
This is the case where an existing backup VDI was found, which will be reused? I think the prompt should make that clearer.